config/csm/test_rest_resource_limits.yaml

```diff
@@ -0,0 +1,8 @@
+test_43149:
+  active_users_quota: 50
+  tmp_users_password: "Seagate@123"
+test_43150:
+  tmp_user_password: "Seagate@123"
+test_43151:
+  requests_rate: 200
+  api_endpoint: "/api/v2/system/users"
```

Comment:
can you use csmuser_endpoint from from cms config

Reply:
updated